secretservice/secretservice_linux.go

```diff
@@ -96,7 +96,8 @@ func (h Secretservice) List() (map[string]string, error) {
 	defer C.freeListData(&acctsC, listLenC)
 	if err != nil {
 		defer C.g_error_free(err)
-		return nil, errors.New("Error from list function in secretservice_linux.c likely due to error in secretservice library")
+		errMsg := (*C.char)(unsafe.Pointer(err.message))
+		return nil, errors.New("Error from list function in secretservice_linux.c likely due to error in secretservice library" + ": " + C.GoString(errMsg))
```

Comment:
just do `return errors.New(C.GoString(errMsg))` like https://github.com/docker/docker-credential-helpers/blob/4b9fe97b79d3d6c1bd77b91bc18b89efd21a7ace/secretservice/secretservice_linux.go#L37 and https://github.com/docker/docker-credential-helpers/blob/4b9fe97b79d3d6c1bd77b91bc18b89efd21a7ace/secretservice/secretservice_linux.go#L53

Reply:
@ebriney Fixed, thanks!